org.jacoco.core/src/org/jacoco/core/data/ExecutionDataWriter.java

```diff
@@ -24,7 +24,11 @@
 		IExecutionDataVisitor {
 
 	/** File format version, will be incremented for each incompatible change. */
-	public static final char FORMAT_VERSION = 0x1007;
+	public static final char FORMAT_VERSION;
+
+	static {
+		FORMAT_VERSION = 0x1007;
+	}
```

Comment:
I can do this by myself, but may I ask you to change back executable bit for this file?
GitHub tells me that old mode was 100644 and new one is 100755:
![mode](https://cloud.githubusercontent.com/assets/138671/21116593/76a25e2e-c0b6-11e6-89b2-ae1c0e8ff79e.png)